pkg/common/deployment.go

```diff
@@ -97,3 +104,28 @@ func (d *Deployment) Delete(
 
 	return nil
 }
+
+// GetDeployment - get the deployment object.
+func (d *Deployment) GetDeployment() appsv1.Deployment {
+	return *d.deployment
+}
+
+// GetDeploymentWithName func
+func GetDeploymentWithName(
+	ctx context.Context,
+	h *helper.Helper,
+	name string,
+	namespace string,
+) (*appsv1.Deployment, error) {
+
+	depl := &appsv1.Deployment{}
+	err := h.GetClient().Get(ctx, types.NamespacedName{Name: name, Namespace: namespace}, depl)
+	if err != nil {
+		if k8s_errors.IsNotFound(err) {
+			return depl, err
+		}
+		return depl, err
```

Comment:
Do we need to check for `IsNotFound` here if both lines 125 and 127 end up returning the same thing?

Reply:
no, will update. thx